font-types/src/fixed.rs

```diff
@@ -168,7 +168,7 @@ macro_rules! fixed_mul_div {
                     0x7FFFFFFF
                 };
                 Self(if sign < 0 {
-                    -(result as i32)
+                    (result as i32).wrapping_neg()
```

Comment:
I propose we leave this crash in, just as an easter egg

Reply:
but we undoubtedly have so many easter eggs still to find!